wurst/systems/spawns/ResourceSpawns.wurst

```diff
@@ -363,3 +418,33 @@ init
         doAfter(25, -> spawnFishAndHawks())
         doAfter(30, -> addItemsToBushes())
         startSpawnCycle()
+
+    // SP for spawning point, Debugging which spot can spawn items
+    registerToolkitCommand("sp") (triggerPlayer, arguments) ->
+        // Look up the index for the triggering player.
+        let index = triggerPlayer.getId()
+        if debug_status[index] == true
+            printTimedToPlayer("Spawning point debug mode disabled".color(GENERAL_COLOR), 10, triggerPlayer)
+            debug_status[index] = false
+        else
+            printTimedToPlayer("Spawning point debug mode enabled, click on a point to check if it's a valid spawning point for items/animals".color(GENERAL_COLOR), 10, triggerPlayer)
+            debug_status[index] = true
+
+        EventListener.add(EVENT_PLAYER_MOUSE_DOWN) ->
+            let x = EventData.getPlayerMouseX()
+            let y = EventData.getPlayerMouseY()
+            let p = EventData.getTriggerPlayer()
+            let point = vec2(x, y)
```

Comment:
Wurst has helper functionality to do this vector directly, `EventData.getMouseWorldPos`.

Reply:
Good to know.